calico/felix/dispatch.py

```diff
@@ -164,16 +165,36 @@ def _calculate_update(self, ifaces):
         # Special case: allow the metadata IP through from all interfaces.
         if self.config.METADATA_IP is not None and self.ip_version == 4:
             # Need to allow outgoing Metadata requests.
-            root_from_upds.append("--append %s "
-                                  "--protocol tcp "
-                                  "--in-interface %s+ "
-                                  "--destination %s "
-                                  "--dport %s "
-                                  "--jump RETURN" %
-                                  (CHAIN_FROM_ENDPOINT,
-                                   self.config.IFACE_PREFIX,
-                                   self.config.METADATA_IP,
-                                   self.config.METADATA_PORT))
+            root_inbound_upds.append("--append %s "
```

Comment:
I think we should just put all of the input rules directly into felix-INPUT - that to me seems like the simplest approach.